appveyor.yml

```diff
@@ -94,6 +94,7 @@ build_script:
       if ($env:GN_CONFIG -eq 'release') {
         python electron\script\zip-symbols.py
         appveyor PushArtifact out/Default/symbols.zip
+        appveyor PushArtifact out/Default/debug.zip
```

Comment:
Only Windows builds are run on AppVeyor. Please remove this line.

Reply:
Done